test/tests/functional/pbs_cgroups_hook.py

```diff
@@ -4093,9 +4094,41 @@ def test_vnodepernuma_use_hyperthreads(self):
         b1 = {'job_state': 'Q'}
         self.server.expect(JOB, b1, jid2)
 
+    def test_cgroup_default_config(self):
+        """
+        Test to make sure using the default hook config file
+        still run a basic job, and cleans up cpuset upon qdel.
+        """
+        # The default hook config has 'memory' subsystem enabled
+        if not self.paths['memory']:
+            self.skipTest('Test requires memory subystem mounted')
+        self.load_default_config()
+        # Reduce the noise in mom_logs for existence=False matching
+        c = {'$logevent': '511'}
+        self.mom.add_config(c)
+        a = {'Resource_List.select': 'ncpus=1:mem=100mb'}
+        j = Job(TEST_USER, attrs=a)
+        j.create_script(self.sleep15_job)
+        jid = self.server.submit(j)
+        a = {'job_state': 'R'}
+        self.server.expect(JOB, a, jid)
+        err_msg = "write_value: Permission denied.*%s.*memsw" % (jid,)
+        self.mom.log_match(err_msg, max_attempts=3, interval=1, n=100,
+                           regexp=True, existence=False)
+        self.server.status(JOB, ['exec_host'], jid)
+        ehost = j.attributes['exec_host']
+        ehost1 = ehost.split('/')[0]
+        ehjd1 = self.get_cgroup_job_dir('cpuset', jid, ehost1)
+        self.assertTrue(self.is_dir(ehjd1, ehost1))
```

Comment:
The second argument to assertTrue() is a message.  That is just a host.  You should turn that into a real message.

Reply:
Ok, I'll add a message.